docs/sails-js/sails-js.md

```diff
@@ -0,0 +1,54 @@
+---
+sidebar_position: 1
+sidebar_label: Sails-JS
+---
+
+# Sails-JS
+
+The `sails-js` library is a powerful tool designed to help developers interact with their Sails applications on Vara Network. It facilitates seamless communication between your TypeScript projects and on-chain programs by providing two primary methods for generating client libraries from a Sails IDL (Interface Definition Language):
```

Comment:
At the next chapter - "The Gear-JS API offers a set of utilities, libraries, and tools that enable JavaScript applications to interact with programs running on the Vara network ..."
Could you pls add some inputs at the beginning about what's the difference between sails-js and gear-js? when and why dapp devs should choose one or another.